services-js/commissions-app/src/pages/commissions/apply.tsx

```diff
@@ -202,6 +205,20 @@ export default class IndexPage extends React.Component<Props> {
   }
 
   renderCommission(commission: Commission) {
-    return <li key={commission.id}>{commission.name}</li>;
+    return (
+      <li style={{ listStyleType: 'none' }}>
+        {' '}
+        <Checkbox
+          name="commissions"
+          value="commission"
+          id={commission.id}
```

Comment:
The `commission.id` here is just a number. The HTML `id` for an element should be more descriptive (`checkbox-commissions-3` or something). I liked when you were generating the `id` inside the `Checkbox` component.